fitbenchmarking/fitbenchmarking
# Add environmental impact table

#### Description of Work

Fixes #1158 

I've added a new emissions table which shows the average CO2 emissions (over `num_runs`) for each minimizer, measured using the [codecarbon](https://mlco2.github.io/codecarbon/index.html) module.

A `.codecarbon.config` file has also been added to set a few parameters which can then be edited by a user if required. Alternatively, the codecarbon parameters can be set when the tracker is initialized.

#### Testing Instructions

1. Run Fitbenchmarking and check that emissions table is produced
2. 
3.

Function: Does the change do what it's supposed to?

Tests: Does it pass? Is there adequate coverage for new code?

Style: Is the coding style consistent? Is anything overly confusing?

Documentation: Is there a suitable change to documentation for this change?
